server/api/controllers/fundraise/create-fundraise.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  friendlyName: 'Create Fundraise Campaign object',
+
+  description: 'Create Fundraise Campaign object',
+
+  inputs: {
+    campaignName: {
+      type: 'string',
+      required: true,
+    },
+    categoryName: {
+      type: 'string',
+      required: true,
+    },
+    categoryKey: {
+      type: 'string',
+      required: true,
+    },
+    amount: {
+      type: 'number',
+    },
+    givers: {
+      type: 'number',
+    },
+    milestones: {
+      type: 'json',
+    },
```

Comment:
had an error when testing api out 
`error: Failed to lift app: userError: The action `fundraise/create-fundraise` could not be registered.  It looks like a machine definition (actions2), but it could not be used to build an action.
Details: ImplementationError: Sorry, could not interpret "fundraise/create-fundraise.js" because its underlying implementation has 4 problems:
------------------------------------------------------
• Invalid input name ("exits").  Please use something else instead, because this is potentially very confusing.
• Invalid input definition ("exits").  Must have `type`, or at least some more information.  (If you aren't sure what to use, just go with `type: 'ref'.)
• Invalid input definition ("fn").  Must be a dictionary-- i.e. plain JavaScript object like `{}`.
• Invalid input definition ("fn").  Must have `type`, or at least some more information.  (If you aren't sure what to use, just go with `type: 'ref'.)`

I think you missed an exit '}' here